main.cpp

```diff
@@ -91,6 +91,129 @@ good to go!
  Wait for my code review.
  */
 
+
+#include <iostream>
+
+//-------------------------------------------------
+// FloatType
+//-------------------------------------------------
+
+class FloatType 
+{
+public:
```

Comment:
use `struct` instead of `class` so you don't have to write `public`
- [ ] revise